hyperlane/cmd/hyp/cmd/broadcaster.go

```diff
@@ -21,13 +22,23 @@ import (
 )
 
 const (
-	mnemonic  = "sphere exhibit essay fancy okay tuna leaf culture elbow drum trip exchange scorpion excuse parent sun make spot chunk mouse tenant shoe hurt scale"
-	chainID   = "celestia-zkevm-testnet"
 	denom     = "utia"
 	feeAmount = 800
 	gasLimit  = 200000
 )
 
+var (
+	mnemonic = getEnvOrDefault("HYP_MNEMONIC", "sphere exhibit essay fancy okay tuna leaf culture elbow drum trip exchange scorpion excuse parent sun make spot chunk mouse tenant shoe hurt scale")
+	chainID  = getEnvOrDefault("HYP_CHAIN_ID", "celestia-zkevm-testnet")
+)
```

Comment:
These seem a bit outside the scope of the PR, but I definitely support these changes regardless so whatever 👍🏻